tests/cases/fourslash/fourslash.ts

```diff
@@ -110,6 +110,7 @@ declare namespace FourSlashInterface {
         bof(): void;
         eof(): void;
         type(definitionIndex?: number): void;
+        implementation(implementationIndex?: number): void;

```

Comment:
The index is never used, and this is always sandwiched between `goTo.marker` and `verify.caretAtMarker`.